hnn_core/cell_response.py

```diff
@@ -476,3 +538,59 @@ def read_spikes(fname, gid_ranges=None):
         cell_response.update_types(gid_ranges)
 
     return cell_response
+
+
+def _read_spikes_hdf5(fname):
+    """Read spiking activity from a hdf5 file.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the hdf5 file (e.g., '<pathname>/spk.hdf5') to be read.
+
+    Returns
+    ----------
+    cell_response : CellResponse
+        An instance of the CellResponse object.
+    """
+
+    data = read_hdf5(fname)
+    cell_response = CellResponse(spike_times=data['spike_times'],
+                                 spike_gids=data['spike_gids'],
+                                 spike_types=data['spike_types'])
```

Comment:
what about `vsec`, `isec` and `times` ?

Reply:
While writing I did store them. While reading them, the parameter list for initializing `cell_response` does not contain `vsec` and `isec`. After initializing an object I tried updating the attribute `vsec` but it threw an error. I can set `_vsec` but this looked wrong as it looked like a private variable...
How should I proceed here?

Reply:
I think you can update `_vsec` but maybe @ntolley do you have a different opinion?

Reply:
This is actually a bug I just ran into today! A really easy way is to try trigger it is to try `net.cell_response[0]` which is supposed to subset to specific gids. 

How about we merge this, and then in the bug fix (opening a PR soon), I'll update this code